doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+.. _quantization:
+
+Quantization
+============
+
+nGraph Quantization 
+-------------------
+
+Quantization refers the process of reducing the number of bits that represent a
```

Comment:
Might want to link to the existing API doc and/or glossary definition that already exists.